src/Razor/test/Microsoft.NET.Sdk.Razor.Test/IntegrationTests/BuildIntrospectionTest.cs

```diff
@@ -142,5 +142,90 @@ public async Task RazorSdk_ResolvesRazorConfigurationToMvc30()
             Assert.BuildOutputContainsLine(result, "RazorLangVersion: 3.0");
             Assert.BuildOutputContainsLine(result, "ResolvedRazorConfiguration: MVC-3.0");
         }
+
+        [Fact]
+        [InitializeTestProject("SimpleMvc")]
+        public async Task UpToDateReloadFileTypes_Default()
+        {
+            var result = await DotnetMSBuild("_IntrospectUpToDateReloadFileTypes");
+
+            Assert.BuildPassed(result);
+            Assert.BuildOutputContainsLine(result, "UpToDateReloadFileTypes: ;.cs;.razor;.resx;.cshtml");
```

Comment:
Without this fix, this value is empty which explains why VS isn't restarting due to changes to cshtml files.